superset-frontend/src/components/Modal.tsx

```diff
@@ -31,7 +31,7 @@ interface ModalProps {
   primaryButtonType?: 'primary' | 'danger';
   show: boolean;
   title: React.ReactNode;
-  bsSize?: 'xs' | 'xsmall' | 'sm' | 'small' | 'medium' | 'lg' | 'large';
+  bsSize?: 'sm' | 'small' | 'lg' | 'large';
```

Comment:
I looked at react-bootstrap's docs because i couldn't believe that both `sm` and `small` were supported for this, but they seem to be.

That said, there's no reason we can't provide a more clear and consistent interface in our Modal wrapper. Maybe we should only support `small` and `large`? The types should still work and it'll ensure one standard way to define this size throughout Superset

Reply:
Sounds perfectly reasonable to me!